src/ReverseProxy/Middleware/AffinitizedDestinationLookupMiddleware.cs

```diff
@@ -66,13 +66,13 @@ public async Task Invoke(HttpContext context)
                         break;
                     case AffinityStatus.AffinityKeyExtractionFailed:
                     case AffinityStatus.DestinationNotFound:
-                        var failureHandled = await _operationLogger.ExecuteAsync("ReverseProxy.HandleAffinityFailure", async () =>
+                        var responseSent = await _operationLogger.ExecuteAsync("ReverseProxy.HandleAffinityFailure", async () =>
                         {
                             var failurePolicy = _affinityFailurePolicies.GetRequiredServiceById(options.AffinityFailurePolicy);
                             return await failurePolicy.Handle(context, options, affinityResult.Status);
                         });
 
-                        if (!failureHandled)
+                        if (!responseSent)
```

Comment:
Clarification: the polarity of responseSent is reversed from failureHandled. If you use this name you'll also need to invert the value returned in the failure policy.

```suggestion
                        if (responseSent)
```

Reply:
Sorry, it's a typo! I actually wanted to use 'keepProcessing' from your suggestion above, but accidentally copied a wrong name. `keepProcessing` has the same polarity as the original `failureHandled`. Will fix the naming.